Flaky CreateAppEngineWtpProjectTest.testUnitTestCreated()

I removed `ProjectUtils.waitForProjects()` in #2054 to see if we are good without it, but seems like we still need it.

```
testUnitTestCreated(com.google.cloud.tools.eclipse.appengine.newproject.standard.CreateAppEngineStandardWtpProjectTest)  Time elapsed: 5.86 sec  <<< FAILURE!
java.lang.AssertionError: null
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at com.google.cloud.tools.eclipse.appengine.newproject.CreateAppEngineWtpProjectTest.assertJunitAndHamcrestAreOnClasspath(CreateAppEngineWtpProjectTest.java:117)
	at com.google.cloud.tools.eclipse.appengine.newproject.CreateAppEngineWtpProjectTest.testUnitTestCreated(CreateAppEngineWtpProjectTest.java:101)
...
```

--- a/plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineWtpProjectTest.java
+++ b/plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineWtpProjectTest.java
@@ -97,6 +97,7 @@ abstract public class CreateAppEngineWtpProjectTest {
   public void testUnitTestCreated() throws InvocationTargetException, CoreException {
     CreateAppEngineWtpProject creator = newCreateAppEngineWtpProject();
     creator.execute(monitor);
+    ProjectUtils.waitForProjects(project);
 
     assertJunitAndHamcrestAreOnClasspath();
   }

